src/Chapter1.hs

```diff
@@ -541,8 +544,9 @@ value after "=" where the condition is true.
 
 Casual reminder about adding top-level type signatures for all functions :)
 -}
-
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid a b c =
+  max (min a b) (min (max a b) c)
```

Comment:
Very clever solution :brain:

Reply:
I wish I could take credit for this but stack overflow provided this answer: https://stackoverflow.com/a/19045659/9089253

Reply:
Well, using SO efficiently is one of the important programming skills. Can't blame you for that!